docs/launching-the-exploit.md

```diff
@@ -85,22 +60,12 @@ If you enter the SD card camera album and nothing unusual happens, ensure that y
 
 :::
 
-You should now see the TWiLight Menu++ language selection.
-
-## Section III - Configuring TWiLight Menu++
-1. When prompted, set your preferred language and region
-   - The default region setting, Game-specific, is generally recommended
-   - After setting them, if the console crashes on white screens, press and hold POWER to power off, power back on, and re-run the exploit
-1. Once you're in TWiLight Menu++, press SELECT to switch to the DS Classic Menu
-1. Tap the DS icon in the bottom middle to open settings
-1. Use the <kbd class="l">L</kbd> / <kbd class="r">R</kbd> or <kbd class="face">X</kbd> / <kbd class="face">Y</kbd> buttons to switch over to the "Misc. settings" page
-1. Switch the "SysNAND Region" entry to your console's region
-1. Press <kbd class="face">B</kbd> to save and exit TWiLight Menu++ settings
+You should now see the TWiLight Menu++ language selection and should select the language and region you prefer to use. They do not need to match your console's language or region.
 
 You are now able to use TWiLight Menu++, however it's recommended to continue on and make a NAND backup. This can potentially be used to save your console if anything bad happens in the future.
```

Comment:
I believe this line was always there, but now it feels janky considering the new line we just added above.